utils/utils.c

```diff
@@ -725,24 +728,28 @@ static int setargs(char *args, char **argv)
 
 		/* consider quotes and update argv */
 		if (*args == QUOTE) {
+			if (argv && *(++args) == QUOTE ) /* skip '' */
+				continue;
```

Comment:
Does it need to check `argv` here?  And I don't want to make the if statement complex.  Also I think the comment can be removed.  What about this?

```c
if (*args == QUOTE) {
   args++;
   if (*args == QUOTE)
      continue;
   ...
```